db/migrate/20170425092225_remove_annee_construction_from_projets.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveAnneeConstructionFromProjets < ActiveRecord::Migration
+  def up
+    Projet.find_each do |projet|
+      demande = projet.demande
+      next unless demande
+      if projet.annee_construction.present?
+        demande.update_attribute(:annee_construction, projet.annee_construction)
+      end
+    end
```

Comment:
C'est généralement une mauvaise idée d'utiliser les modèles dans une migration (pour les raisons [détaillées ici](http://guides.rubyonrails.org/v4.1/migrations.html#using-models-in-your-migrations)). On a déjà une migration qui a pété en démo et en prod à cause de ça.

_For the record_, je pense qu'on devrait plutôt utiliser une tâche rake exécutée après les migrations (par exemple avec [after_party](https://github.com/theSteveMitchell/after_party)). Mais c'est comme tu le sens :)